dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/Execution.java

```diff
@@ -39,12 +39,12 @@ public class Execution {
     private String executionTime;
 
     @JsonProperty
-    @Schema(description = "Memory requirements for the execution in GB. Provide the numerical value and the GB unit with no spaces", example = "2GB")
+    @Schema(description = "Memory requirements for the execution in GB. Provide the numerical value and the GB unit separated by a space", example = "2 GB")
```

Comment:
Miscellaneous rambling: I have mixed feelings about requiring units with the inputs.  On the plus side, the units are tied to the quantity, there's no ambiguity about what the number means.  On the minus side, they invite various sorts of alternate non-kosher forms ("2", "2G", "2GB", "2 gb", "2 MB") and complicate the processing.  No action necessary here...

Reply:
I briefly had the same thought, but didn't address it because I wasn't sure if it was worth it. Since you also had the same thought, maybe it is :slightly_smiling_face: 

Perhaps it would be better to change `String memoryRequirements` to `Double memoryRequirementsInGB`? That way the user would just input the numerical value and we'll assume it's in GB. It would simplify the aggregation because we won't need to parse a string.